components/fxa-client/src/account.rs

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! # Account Management URLs
+//!
+//! Signed-in applications should not attempt to perform an account-level management
+//! (such as changing profile data or managing devices) using native UI. Instead, they
+//! should offer the user the opportunity to visit their account management pages on the
+//! web.
+//!
+//! The methods in this section provide URLs at which the user can perform various
+//! account-management activities.
+
+use crate::{ApiResult, Error, FirefoxAccount};
+use error_support::handle_error;
+
+impl FirefoxAccount {
+    /// Get the URL at which to access the user's sync data.
```

Comment:
heh - this comment kinda lies as it skips a level of indirection - It "Get the URL at which the URL and access token to access the user's sync data can be obtained" :)

(I don't care if you don't update it as I'm sure you just copied it)